dev/parm/config/gfs/yaml/defaults.yaml

```diff
@@ -16,7 +16,7 @@ base:
   DO_GENESIS_FSU: "NO"
   DO_METP: "YES"
   FHMAX_GFS: 120
-  FHMAX_HF_GFS: 0
+  FHMAX_HF_GFS: 48
```

Comment:
Why change the default to hourly up to 48hrs?

Reply:
so that we can ensure that the `HF` is actually being tested.

Reply:
Good thinking!